apps/studio/src/lib/db/clients/bigquery.js

```diff
@@ -126,6 +224,11 @@ function query(client, queryText) {
   }
 }
 
+async function applyChanges(conn, changes) {
```

Comment:
Should this return something?

Reply:
Also still needs fixing

Reply:
We said we weren't doing write mode for this release, I can have it throw an error or something if you'd like?

Reply:
Sure